lodash.src.js

```diff
@@ -7325,25 +7325,17 @@
      * _.map(_.sortByAll(users, ['user', 'age']), _.values);
      * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
      */
-    function sortByAll() {
```

Comment:
Super nice cleanup of this method. `_.restParam` is great for making things more readable!